src/web/experiments/tests/newsletter-merch-unit-test.ts

```diff
@@ -0,0 +1,49 @@
+import { ABTest } from '@guardian/ab-core';
+
+export const newsletterMerchUnitLighthouseControl: ABTest = {
+    id: 'NewsletterMerchUnitLighthouseControl',
+    start: '2020-11-11',
+    expiry: '2020-12-01',
+    author: 'Josh Buckland & Alex Dufournet',
+    description: 'Show any merch unit to 50% of users. This is the control for the NewsletterMerchUnitLighthouseVariants test.',
+    audience: 0.5,
+    audienceOffset: 0.0,
+    successMeasure: 'We see increased engagement from users shown the Newsletters ad unit',
+    audienceCriteria: 'Website users only.',
+    idealOutcome: 'Investigate lighthouse segment engagement via newsletters',
+    showForSensitive: false,
+    canRun: () => true,
+    variants: [
+        {
+            id: 'control',
+            test: (): void => {},
+        },
+    ],
+};
+
+export const newsletterMerchUnitLighthouseVariants: ABTest = {
+    id: 'NewsletterMerchUnitLighthouseVariants',
+    start: '2020-11-11',
+    expiry: '2020-12-01',
+    author: 'Josh Buckland & Alex Dufournet',
+    description: 'Show a newsletter advert in the merchandising unit to 25% of users. ' +
+        'These two variants test value of showing newsletter merch units instead of reader revenue ones. ' +
+        'This test needs to run at the same time as NewsletterMerchUnitLighthouseVariants',
```

Comment:
I think this is meant to be
```suggestion
        'This test needs to run at the same time as NewsletterMerchUnitLighthouseControl',
```

Reply:
Thanks